### DISCUSSION TITLE
Process is not defined inside boot file

### DISCUSSION THREAD

**Discussion by DevNick2** [quasarframework/quasar#10347] [CLI - PWA mode]
Hello everyone.

So, I read many discussion in forum about how using process.env on quasar v2 but in my case isn't working.

My quasar.conf.js:

![image](https://user-images.githubusercontent.com/30379011/129016522-917a8fc4-4a74-460d-a7d1-9aff1fa9733b.png)

My axios.js boot file:

![image](https://user-images.githubusercontent.com/30379011/129016645-3fe3c70f-04f5-42ab-a937-1788d274780e.png)

Feedback in console:

![image](https://user-images.githubusercontent.com/30379011/129016695-78935d6e-7f19-451e-9683-ea1ccd2f4236.png)

Thanks for all :D

**Comment by ontwikkelfabriek** [comment]
@DevNick2 

Does  `process.env.API` work in Vue files somewhere else in your app? (test it somewhere in the script section, just try)

**Reply by DevNick2** [reply]
Hello @ontwikkelfabriek, thanks for answer my question.

I belive that cache is responsible for process.env not work in my project.

But, I tried in some Vue file on script section and not working.

(Sorry for my bad english :D)

**Reply by ontwikkelfabriek** [reply]
@DevNick2 

I would create a new clean default Quasar project, with Quasar CLI , with same Quasar version as your own project (Qv1 or Qv2)

Then see it does work in the new default project.

**Comment by adamlytics** [comment]
I just created a brand new project with the latest version of Quasar.  I am trying to use the `process` variable inside of the axios boot file and it is not working there.  I can access it in the boot files when defining data.  I have an older Quasar project where this does work.

**Reply by adamlytics** [reply]
I've also found this to be an issue with other libraries that for some reason call the process variable.  I'm using vue-stripe, and someone with a Quasar app has reported having an error where `process is not defined` when trying to use this plugin.  I'm seeing this as well and think it is related to this issue.

**Reply by DevNick2** [reply]
Hello @adamlytics,

Thanks for your anwser, I found the solution, it's very crazy but, when a run command quasar build -m pwa in my environment, the chunks was in public folder, and my browser was geting files of my public folder, and this was the problem.

My solution was, remove all things of my public folder, rebuild the icongenie and run quasar dev -m pwa, this is working very well.

Thanks to all answers
